### PR TITLE
Explore default-limits LimitRange

### PR DESCRIPTION
**Pull request by mtspn**

### CODE REVIEW

**Review by ty2k — CHANGES_REQUESTED** [review]
The code blocks are a little broken in this commit. Here's the [docs for how code blocks work in Markdown on GitHub](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks), but the TLDR is when using the three backticks style of code fence, any text to the right of the first three backticks is used to reference the language of the code block for syntax highlighting on the rendered HTML page. Instead, we need a linebreak after the first three backticks, so the code we want to display is between two "fences" of three backticks. For inline code blocks, they only need to be surrounded by one backtick, and the inline code shouldn't have any line breaks.

File: 101-lab/content/05_resource_management.md [hunk 1]
```diff
@@ -9,7 +9,15 @@ Many templates include reasonable CPU and Memory resource configurations, howeve
...
-- Take a look a the resources spec under `.spec.template.spec.containers.resources`. These values should reflect the resource requests (cpu=500m,memory=512Mi) and limits (cpu=1000m,memory=1024Mi) we set earlier.  If we had not set any values for requests or limits earlier, we would see `{}` in the resources spec to indicate the deployment was using the default resource allocation. 
+- Take a look a the resources spec under `.spec.template.spec.containers.resources`. These values should reflect the resource requests (cpu=500m,memory=512Mi) and limits (cpu=1000m,memory=1024Mi) we set earlier.  If we had not set any values for requests or limits earlier, we would see `{}` in the resources spec to indicate the deployment was using the default resource allocation for the platform. 
+
+- To view the limitranges that are in place, run the command: 
+```oc get LimitRange
+```
+
+- You should see there is a limitrange in place called 'default-limits'. This set of request and limit values are applied when resource specifications are not defined.  To view the details of this limitrange, run: 
+```oc -n [-dev] describe LimitRange default-limits
```
Comment (mtspn) [inline 101-lab/content/05_resource_management.md:19]:
```suggestion
```
oc -n [-dev] describe LimitRange default-limits
```

File: 101-lab/content/05_resource_management.md [hunk 2]
```diff
@@ -9,7 +9,14 @@ Many templates include reasonable CPU and Memory resource configurations, howeve
...
 - Navigate to your rocketchat deployment and select `Yaml`
-- Take a look a the resources spec under `.spec.template.spec.containers.resources`. These values should reflect the resource requests (cpu=500m,memory=512Mi) and limits (cpu=1000m,memory=1024Mi) we set earlier.  If we had not set any values for requests or limits earlier, we would see `{}` in the resources spec to indicate the deployment was using the default resource allocation. 
+- Take a look a the resources spec under `.spec.template.spec.containers.resources`. These values should reflect the resource requests (cpu=500m,memory=512Mi) and limits (cpu=1000m,memory=1024Mi) we set earlier.  If we had not set any values for requests or limits earlier, we would see `{}` in the resources spec to indicate the deployment was using the default resource allocation for the platform. 
+
+- To view the limitranges that are in place, run the command: 
+```oc get LimitRange
+```
+
+- You should see there is a limitrange in place called 'default-limits'. This set of request and limit values are applied when resource specifications are not defined.  To view the details of this limitrange, run: 
+```
```
Comment (mtspn) [inline 101-lab/content/05_resource_management.md:19]:
```suggestion
- You should see there is a limitrange in place called 'default-limits'. This set of request and limit values are applied when resource specifications are not defined.  To view the details of this limitrange, run: 
```
```oc -n [-dev] describe LimitRange default-limits
```
```

**Comment by mtspn** [comment]
I've fixed the code blocks, that'll teach me to always check the rich preview!

**Pull request merged**